kuka_lbr_iiwa_support/launch/test_lbr_iiwa_14_r820.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
+  <include file="$(find kuka_lbr_iiwa_support)/launch/load_lbr_iiwa_14_r820.launch" />
+  <param name="use_gui" value="true" />

```

Comment:
'new style' test launch files put this parameter in the private namespace of the `joint_state_publisher` node.